docs/src/lib/transform.ts

```diff
@@ -49,6 +48,23 @@ function transformFunction({
   return { id, name, description, methods, sourceUrl };
 }
 
+function transformSummary(
+  summary: Array<JSONOutput.CommentDisplayPart>,
+): string | undefined {
+  if (summary.length === 0) {
+    return undefined;
+  }
+  return summary
+    .map((part) => {
+      const { kind, text } = part;
+      if (kind === "inline-tag" && part.tag === "@link") {
+        return `[\`${text}\`](#${text})`;
```

Comment:
It's hard to notice the little hash sign before the text for the link; extract a function for building the URI and add some documentation to describe what's going on. Also, we probably don't want to completely block the ability to add links to external sites, perhaps we should check that the text doesn't start with any URL prefix (like / or https://) and only add the hash when it doesn't.

```ts
function linkHref(raw: string): string {
  if (raw.startsWith("http") || raw.startsWith("/")) {
    return raw;
  }
  // We assume that any simple link is a reference to a function...
  return `#${raw}`;
}
```

```suggestion
       return `[\`${text}\`](${linkHref(text)})`;
```

Reply:
if we wanted this to be a bit safer, we can pass through `target`; i made this change